src/test/java/co/rsk/federate/signing/hsm/message/AdvanceBlockchainMessageTest.java

```diff
@@ -82,4 +84,99 @@ public void test_getParsedBrothers_invalid_blockHeader() throws HSMBlockchainBoo
 
         message.getParsedBrothers(Hex.toHexString(invalidBlockHeader.getFullEncoded()));
     }
+
+    @Test
+    public void test_getParsedBrothers_sorted_by_hash() throws HSMBlockchainBookkeepingRelatedException {
+        BlockHeader block1Header = blockHeaderBuilder.setNumber(1).build();
+        BlockHeader block2Header = blockHeaderBuilder.setNumber(2).build();
+
+        List<BlockHeader> block1Brothers = Arrays.asList(
+            TestUtils.createBlockHeaderMock(102),
```

Comment:
Why go back to using mocks instead of BlockHeaderBuilder?

Reply:
Updated